www/_posts/2017-12-04-cordova-android-7.0.0.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+author:
+    name: Joe Bowser
+    url: https://twitter.com/infil00p
+title:  "Cordova Android 7.0.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that `Cordova Android 7.0.0` has been released! 
+
+With this release, we have changed the default project structure for Android projects.  People who currently use the CLI and treat everything in the platforms directory as a 
+build artifact should not notice a difference, however this a major breaking change for people creating standalone Cordova Android projects.  This also means that the locations
+of files have changed and have been brought in line to the structure used by Android Studio.
```

Comment:
We probably need to add something here to tell people that `edit-config` tags in plugin.xml and config.xml will need their paths updated?

i.e.,
```xml
<!-- An existing config.xml -->
<edit-config file="AndroidManifest.xml" target="/manifest/application" mode="merge">

<!-- needs to change to -->
<edit-config file="app/src/main/AndroidManifest.xml" target="/manifest/application" mode="merge">
```

Reply:
That's a good idea.

Reply:
Actually, if you search GitHub for `edit-config`, it seems like 99% of them are editing AndroidManifest.xml. Is it possible to special case that so it uses the right path? Did that already happen in cordova-common and we're waiting for a tools release?

Reply:
I don't believe any of edit-config's functionality has been changed.  Of course, we only test on our own plugins, and none of those use edit-config on Android, only on iOS.  I'm sure it's possible to add this special case,  but that would mean that we would be on the hook for maintaining it, when people really should be editing the path.  I think we wrote enough new workaround code for this change already.

Reply:
Then we should also mention that it's not possible for a plugin using `edit-config` to support both cordova-android@6 and cordova-android@7?

Reply:
Fair enough.  That said, what's an example plugin that is widely used that would be broken by this?

Reply:
CSDK plugins are affected by this but they are not being actively maintained.

Reply:
https://github.com/macdonst/cordova-plugin-allow-backup is one, I know I've used edit-config in a few of my company's internal plugins

Reply:
@dpogue no one needs to use cordova-plugin-allow-backup as they can just add those lines in config.xml instead.

Reply:
OK, I added a brief mention after the first paragraph and copy-pasted your code example.  BTW: We can all edit the PR in GitHub itself.